package/tests/test_models/test_flowchart.py

```diff
@@ -228,6 +229,20 @@ def test_create_non_existing_parameter_node(self, patched_warning):
             [call("Cannot find parameter `%s` in the catalog.", "non_existing")]
         )
 
+    @patch("logging.Logger.warning")
```

Comment:
Should we also test the `AttributeError` case (presumably caused by `self.kedro_obj = None`)? Just a thought - up to you.

Reply:
Good thought, this case is covered by the test directly above 🙂. I added comments to both tests clarifying their purpose in [786d23e](https://github.com/kedro-org/kedro-viz/pull/1231/commits/786d23e0f1cb3ad2d442a56153e619a7097d9e3e).